content/react.md

```diff
@@ -180,7 +180,9 @@ React is a front-end rendering library and as such doesn't concern itself with h
 
 To integrate the two systems, we've developed a [`react-meteor-data`](https://atmospherejs.com/meteor/react-meteor-data) package which allows React components to respond to data changes via Meteor's [Tracker](https://www.meteor.com/tracker) reactivity system.
 
-<h3 id="using-createContainer">Using `createContainer`</h3>
+<h3 id="using-withTracker">Using `withTracker`</h3>
+
+> The `withTracker` function now replaces the previous function `createContainer`, however it remains as part of the package for backwards compatibility.
```

Comment:
This is a comma splice. Replace the comma with a semicolon or make two sentences.

Reply:
Maybe we change this to something like:

> The `withTracker` function replaces the previously recommended `createContainer` function. The  `createContainer` function can still be used, but will be officially deprecated in the not too distant future.